ux.symfony.com/assets/styles/components/_Terminal.scss

```diff
@@ -170,15 +172,15 @@
     }
 }
 
-@media screen and (min-width: 768px) {
-    .Terminal_content::-webkit-scrollbar {
-        display: none;
-    }
-    .Terminal_content {
-        --webkit-scrollbar-width: none;
-        scrollbar-width: none;
-    }
-}
+// @media screen and (min-width: 768px) {
+//     .Terminal_content::-webkit-scrollbar {
+//         display: none;
+//     }
+//     .Terminal_content {
+//         --webkit-scrollbar-width: none;
+//         scrollbar-width: none;
+//     }
+// }
```

Comment:
I'm not sure about this one, but I don't really see why you would want to hide a scrollbar, as it reduce accessibility and UX

Reply:
Is this related to Toolkit ? Any page i can  see the problem ?

Reply:
Yes and no, I wanted to display a scroll bar when necessary, but I couldn't understand why this code exists.
You can check the "Alert" component page, by unfolding the files from "manual installation" section, it contains a Terminal with a veryyy long and unbreakable line.

You can also check other parts of the website where a Terminal is displayed, to confirm it didn't break things. 🙏

Reply:
Because it broke things at a time, two scrollbars were displayed. Testing on mac often gives a false impression, as macOS hide scrollbars per default.

But let's see and we will be able to fix if there were any pb 

👍 :